android/src/main/java/com/reactnativekeyboardcontroller/views/overlay/JSPointerDispatcherCompat.kt

```diff
@@ -0,0 +1,36 @@
+package com.reactnativekeyboardcontroller.views.overlay
+
+import android.view.MotionEvent
+import android.view.ViewGroup
+import com.facebook.react.uimanager.JSPointerDispatcher
+import com.facebook.react.uimanager.events.EventDispatcher
+
+/**
+ * Compat layer for `JSPointerDispatcher` interface for RN < 0.72
+ */
+class JSPointerDispatcherCompat(
+  private val viewGroup: ViewGroup,
+) : JSPointerDispatcher(viewGroup) {
+  fun handleMotionEventCompat(
+    event: MotionEvent?,
+    eventDispatcher: EventDispatcher?,
+    isCapture: Boolean,
+  ) {
+    val method =
+      try {
+        JSPointerDispatcher::class.java.getMethod(
+          "handleMotionEvent",
+          MotionEvent::class.java,
+          EventDispatcher::class.java,
+          Boolean::class.javaPrimitiveType,
+        )
+      } catch (e: NoSuchMethodException) {
```

Comment:
I think `detekt` is complaining about this - we are writing `catch`, but `e` is unused (so we swallow an exception)

I think we can simply rename it to `_` (since we have only two cases at the moment) and detekt will be happy 🙂

Reply:
thx !